app/src/main/java/com/example/photostomusic/ColorFormActivity.java

```diff
@@ -0,0 +1,186 @@
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.parse.GetCallback;
+import com.parse.ParseException;
+import com.parse.ParseObject;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+// This activity will hold the color form to be filled out by users to map their feelings
+public class ColorFormActivity extends AppCompatActivity {
+
+    public final String TAG = this.getClass().getSimpleName();
+
+    // Visual elements of the activity
+    Button btnConfirmColors;
+    AppCompatSpinner redSpinner;
+    AppCompatSpinner violetSpinner;
+    AppCompatSpinner blueSpinner;
+    AppCompatSpinner greenSpinner;
+    AppCompatSpinner yellowSpinner;
+    AppCompatSpinner orangeSpinner;
+
+    // Temp list used to test dropdowns
+    List<String> emotions = new ArrayList<String>(Arrays.asList(
+            "default", "Happy", "Tranquil", "Scared", "Amazed",
+            "Sad", "Tired", "Angry", "Alert"));
+
+    // List used to contain all currently selected options
+    Set<String> usedOptions = new HashSet<String>();
+
+    // List used to hold all previously selected items
+    AdapterView.OnItemSelectedListener genericListener;
+
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_color_form);
+
+        // Connect visual and logic elements
+        btnConfirmColors = findViewById(R.id.btnConfirmColors);
+        redSpinner = findViewById(R.id.redSpinner);
+        violetSpinner = findViewById(R.id.violetSpinner);
+        blueSpinner = findViewById(R.id.blueSpinner);
+        greenSpinner = findViewById(R.id.greenSpinner);
+        yellowSpinner = findViewById(R.id.yellowSpinner);
+        orangeSpinner = findViewById(R.id.orangeSpinner);
+
+        // Add listener to confirm button, set a RESULT_OK code for the activity if the form is valid
+        btnConfirmColors.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                HashMap<String, String> map = new HashMap<String, String>() {{
+                    put("FF0000", redSpinner.getSelectedItem().toString());
+                    put("634598", violetSpinner.getSelectedItem().toString());
+                    put("2196F3", blueSpinner.getSelectedItem().toString());
+                    put("4CAF50", greenSpinner.getSelectedItem().toString());
+                    put("FFEB3B", yellowSpinner.getSelectedItem().toString());
+                    put("FF9800", orangeSpinner.getSelectedItem().toString());
```

Comment:
again, having these values as variables would make it a lot easier to read and it would make it easier on you. if you want to tweek a color a bit, you would only have to change the variable instead of finding every instance of that hex value